bins/recipe-runner/src/runner.c

```diff
@@ -631,7 +638,12 @@ GglError runner(const RecipeRunnerArgs *args) {
         &resp
     );
     switch (ret) {
+    case GGL_ERR_NOMEM:
+        GGL_LOGW("Failed to get noProxyAddresses from config - lack of memory."
+        );
+        break;
     case GGL_ERR_NOENTRY:
+        GGL_LOGW("Failed to get noProxyAddresses - no such entry.");
```

Comment:
Same as above, this is the expected happy-path.

Reply:
Done